src/test/java/accounts/OpenAccountPageTest.java

```diff
@@ -2,18 +2,30 @@
 
 import base.BaseTest;
 import data.UserData;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
+import pages.AccountPage;
 import pages.HomePage;
 import pages.LogInPage;
 
+import static org.testng.Assert.assertEquals;
+
 public class OpenAccountPageTest extends BaseTest {
     public LogInPage logInPage;
     public HomePage homePage;
+    public AccountPage accountPage;
 
-    @Test
-    public void openAccountPageTest() throws InterruptedException {
+    @BeforeTest
+    public void Login() throws InterruptedException {
         logInPage = base.openPlayGroundPage();
         homePage = logInPage.logInPlayGround(UserData.ADMIN_USER_EMAIL, UserData.ADMIN_USER_PASSWORD);
         Thread.sleep(5000);
```

Comment:
Don't use Thread.sleep. Try to write explicitly wait and use it not into the test but on the page classes.

Reply:
@Evangard done